tests/ut_mcalendar/ut_mcalendar.cpp

```diff
@@ -5726,14 +5731,14 @@ void Ut_MCalendar::testWeekdayType_data()
        << "de_DE"
        << MLocale::DefaultCalendar
        << 1
-       <<  0 <<  0 <<  0 <<  0 <<  0 << 1 << 3
+       <<  0 <<  0 <<  0 <<  0 <<  0 << 1 << 1 // sunday is weekend if transition occurs at midnight
```

Comment:
Again these are "not expecting anything special" data which now changes expectation.